projects/driver_controls/inc/input_event.h

```diff
@@ -24,5 +24,9 @@ typedef enum {
   INPUT_EVENT_MECHANICAL_BRAKE_RELEASED,
   INPUT_EVENT_HORN,
   INPUT_EVENT_PUSH_TO_TALK,
+  INPUT_EVENT_HEADLIGHT,
```

Comment:
Please order these by priority since event queue is a priority queue and BMS fault is probably high priority?

Reply:
Done